lightning_pose/models/heatmap_tracker.py

```diff
@@ -142,9 +127,10 @@ def __init__(
         self.rmse_loss = RegressionRMSELoss()
 
         # necessary so we don't have to pass in model arguments when loading
-        # added loss_factory_unsupervised which might come from the SemiSupervisedHeatmapTracker.__super__(). Otherwise it's ignored.
+        # added loss_factory_unsupervised which might come from the
+        # SemiSupervisedHeatmapTracker.__super__(). Otherwise it's ignored.
         # that's important so that it doesn't try to pickle the dali loaders.
-        self.save_hyperparameters(ignore=["loss_factory", "loss_factory_unsupervised"])  # cannot be pickled
```

Comment:
i think we should keep the "cannot be pickled" because we can run into issues if we don't do that